Extensions/move-blog-selector.js

```diff
@@ -0,0 +1,51 @@
+//* TITLE Blog Selector to bottom **//
+//* VERSION 1.0.0 **//
+//* DESCRIPTION Moves the blog selector from top of form to bottom of form	**//
+//* DEVELOPER manavortex **//
+//* FRAME false **//
+//* BETA false **//
+
+XKit.extensions.moveblogselector = new Object({
+	
+	running: false,
+	
+	run: function() {
+		this.running = true;
+		this.waitForAndPrepend(".post-form--header", ".post-form--bottom");
+	},
+	
+	destroy: function() {
+		this.running = false;
+	}, 
+	
+	/**		
+	* Async function: call with checkElement('#myElement').then((el) { ... }) .
+	*
+	* @param {string} selector - document query selector.
+	*/
+	checkElement: async function(selector) {
+		while ( this.running && document.querySelector(selector) === null) {
```

Comment:
not sure how eslint didn't catch this, but you should remove the spaces on the inside of parentheses here
```suggestion
		while (this.running && document.querySelector(selector) === null) {
```